packages/api/lib/ems.js

```diff
@@ -101,6 +103,22 @@ async function getExpiredS3Objects(bucket, prefix, retentionInDays) {
     .map((s3Object) => ({ Bucket: bucket, Key: s3Object.Key }));
 }
 
+async function retrievePrivateKey() {
+  const privateKeyFile = process.env.ems_privateKey || 'ems-private.pem';
+
+  const keyExists = await s3ObjectExists(
+    { Bucket: process.env.system_bucket, Key: `${process.env.stackName}/crypto/${privateKeyFile}` }
+  );
+  if (!keyExists) {
+    return Promise.reject(new Error(`${privateKeyFile} does not exist in S3 crypto directory`));
+  }
+
+  log.debug(`Reading Key: ${privateKeyFile} bucket:${process.env.system_bucket}, stack:${process.env.stackName}`);
+  const privateKey = await getS3Object(process.env.system_bucket, `${process.env.stackName}/crypto/${privateKeyFile}`);
```

Comment:
You can use the [getTextObject()](https://github.com/nasa/cumulus/blob/master/packages/aws-client/S3.js#L281) function here:

```js
return getTextObject(
  process.env.system_bucket,
  `${process.env.stackName}/crypto/${privateKeyFile}`
);
```

Reply:
Cool I didn't know that existed. Thanks!